Verifications after chectl installation (#806)
codenvy/che-docs: src/main/pages/che-7/overview/proc_installing-the-chectl-management-tool-on-linux-or-macos.adoc

> . Run the following commands in the terminal:

Reviewer: ```suggestion
. Run the following command in the terminal (this downloads and executes the `install.sh` script):
```

Reviewer: The prereqs for this procedure should include the necessity to have the `sudo` command (environment) configured for the current user. (This isn't configured by default on RHEL, for example.)

+
----

> $ bash <(curl -sL  https://www.eclipse.org/che/chectl/)

Reviewer: Maybe use just `sh` to make it more universal?
```suggestion
$ sh <(curl -sL  https://www.eclipse.org/che/chectl/)
```

Reviewer: Ah, I see the script itself uses `bash`. Ok, disregard.

----
+
[NOTE]
====

> The installation script requires superuser access.

Reviewer: The above comment about the `sudo` prereq should make this admonition redundant.

> If necessary the user will be prompted for his password by sudo.

Reviewer: @themr0c, just a docs note: avoid 'if' and 'will' :)

====
+

> . Run the following command to verify that the `chectl` binary that is used is `/usr/local/bin/chectl`:

Reviewer: ```suggestion
. Run the following command to verify that the `chectl` binary used is `/usr/local/bin/chectl`:
```

+
----
$ which chectl
/usr/local/bin/chectl
----
+

> [TIP]

Reviewer: This step should be before the installation of the right `chectl`:

1. (prereq) Remove all old or undesired versions of `chectl`
2. Run the install script
3. Verify that the install `chectl` is in your path

====
If the output of the `which chectl` command is not `/usr/local/bin/chectl`, it probably means that an old version of chectl is present in a place with higher priority in your $PATH. If this other version appear to be unnecessary, run the following command to delete it:

----
# rm $(which chectl)
----
====

. Run the following command to verify that the version of chectl that is the expected one.
+
----
$ chectl --version
----
+
[TIP]
====

> To determine the lates stable release navigate to the  link:https://github.com/che-incubator/chectl/releases[list of Chectl releases] and search for a release not containing "next" in her name.

Reviewer: ```suggestion
To identify the latest stable release, see the link:https://github.com/che-incubator/chectl/releases[list of chectl releases] and search for a release not containing "next" in the name.
```

====

. Read the installation logs.
+
----

> cat chectl-install.log

Reviewer: ```suggestion
$ cat chectl-install.log
```
